contracts/cw20-staking/src/contract.rs

```diff
@@ -84,41 +82,47 @@ pub fn handle<S: Storage, A: Api, Q: Querier>(
         HandleMsg::_BondAllTokens {} => _bond_all_tokens(deps, env),
 
         // these all come from cw20-base to implement the cw20 standard
-        HandleMsg::Transfer { recipient, amount } => handle_transfer(deps, env, recipient, amount),
-        HandleMsg::Burn { amount } => handle_burn(deps, env, amount),
+        HandleMsg::Transfer { recipient, amount } => {
+            Ok(handle_transfer(deps, env, recipient, amount)?)
+        }
+        HandleMsg::Burn { amount } => Ok(handle_burn(deps, env, amount)?),
         HandleMsg::Send {
             contract,
             amount,
             msg,
-        } => handle_send(deps, env, contract, amount, msg),
+        } => Ok(handle_send(deps, env, contract, amount, msg)?),
```

Comment:
If `handle_send`, etc all return `Result<HandleResponse, ContractError>`, not `StdResult<HandleResponse>`, then we don't need the `Ok(handle_send(..)?)` idiom do we? that is just when we need to auto-convert error cases.

Reply:
Yes, AFAIK.